app/views/datasets/topics/_errors.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if @dataset.errors.any? %>
```

Comment:
This file doesn't look specific to topics - should it go one level up so it's more likely to be re-used?

Reply:
Yes it should. I created a partial here to remove duplication between the new and edit views. However there seems to be a lot of duplication between the various forms for topics, license etc that would be a good candidate to refactor out at a later date.